src/json/json.cc

```
@@ -289,11 +289,11 @@ STATIC JsonUtilCode parseAndValidateMSetCmdArgs(ValkeyModuleCtx *ctx, ValkeyModu
     JsonUtilCode rc;
 
     // Allocate memory for args_list using a unique_ptr
-    std::unique_ptr<MSetCmdArgs> args_list((MSetCmdArgs*)ValkeyModule_Alloc(num_keys * sizeof(MSetCmdArgs)));
+    std::unique_ptr<MSetCmdArgs[]> args_list(new (std::nothrow) MSetCmdArgs[num_keys]);
```

why using std::nothrow?

---

It will return a null pointer if the allocation fails.

---

That doesn't make sense. If the allocation fails, we should abort, instead of hiding the failure.

---

Yes, This is a miss in this implementation which I can add